agent_application/docker/docker-compose.yml

```diff
@@ -34,5 +34,7 @@ services:
     environment:
       AGENT_CONFIG_LOG_FORMAT: json
       AGENT_CONFIG_EVENT_STORE: postgres
+      # Relative paths can't start or end with '/'
+      AGENT_RELATIVE_PATH: "unicore"
```

Comment:
A common naming for this is `BASE_PATH`. Do you wanna update to `AGENT_APPLICATION_BASE_PATH`?